src/libs/actions/Report.js

```diff
@@ -521,6 +521,47 @@ function openPaymentDetailsPage(chatReportID, iouReportID) {
     });
 }
 
+/**
+ * Gets transactions and data associated with the linked report (expense or IOU report)
+ *
+ * @param {String} chatReportID
+ * @param {String} linkedReportID
+ */
+function openMoneyRequestsReportPage(chatReportID, linkedReportID) {
+    API.read('OpenMoneyRequestsReportPage', {
+        reportID: chatReportID,
+        linkedReportID,
+    }, {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.IOU,
+                value: {
+                    loading: true,
```

Comment:
NAB, I don't think it needs to be done now. But anyone else think it's weird that we have an `IOU` key that only exists to set a `loading: Boolean` state?

Reply:
Yea, this was pretty confusing to me. I don't think we should tackle it as part of this PR, but this is definitely something that we should consider refactoring.

Reply:
Yes, I felt the same but I just wanted to keep the scope simple and mimic what we had for the OpenPaymentsDetails page.